assignments/this.js

```diff
@@ -13,14 +13,61 @@
 
 // code example for Window Binding
 
+var sayMyName = function () {
+    `use strict`;
+    console.log(this.name);
+  };
+  
+  var me = {
+    name: 'York'
```

Comment:
Why use var? The rule of thumb is used `const` until it throws an error then use `let`. If you EVER need to use `var` you will know it.